content/tutorials/xgboost_deployment.md

```diff
@@ -0,0 +1,108 @@
++++
+title= "XGBoost Deployment"
+date= 2018-03-12T17:08:30-07:00
+description = "Guide to deploy XGBoost Models."
+draft= false
++++
+## A guide to installing packages to model containers
+{{% notice warning %}}
+This tutorial will assume that you have already installed `clipper_admin` and its dependencies.
+{{% /notice %}}
+
+In order to keep the base image's light, containers limit the packages they come with default; however, Clipper also supports the installation of additional packages onto model containers using pip. Packages such as XGBoost can be installed and deployed in this way.
+
+First, we open a python interactive shell.
+```sh
+$ python
```

Comment:
I think for this tutorial, it makes sense to just show the Python commands, not show the full shell session with output (despite the fact that the quickstart does this). This will make it easier for people to copy and paste.

Reply:
Cool! I was just trying to follow the Quickstart example (as you figured out), but I can change the style.

Reply:
I did include the output for the very last block where it returns a prediction, because I thought that might be nice for people to see.